agent/agentbootstrap/bootstrap.go

```diff
@@ -165,6 +173,22 @@ func InitializeState(
 		return nil, nil, errors.Annotate(err, "cannot initialize bootstrap machine")
 	}
 
+	if hasHostedModel {
```

Comment:
We want the same behaviour as with IAAS - a "default" model created

Reply:
yes, added `default` back.